### PR TITLE
implemented algorithm for matches page

### PR DESCRIPTION
**Pull request by jpnoe**

### CODE REVIEW

**Review by lalvessiqueira — COMMENTED** [review]
Great work!

**Review by lalvessiqueira — COMMENTED** [review]
delete node_modules

File: backend/app.py [hunk 1]
```diff
@@ -132,6 +145,15 @@ def not_found(error=None):
     resp = jsonify(message)
     resp.status_code = 404
     return resp
+    
+def stringify_userid(user):
+    user["_id"] = str(user["_id"])
+    return user
+    
+def get_scores(x, user):
```
Comment (ericta-portfolio) [inline backend/app.py:153]: x could have a better name

File: backend/compareProfiles.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+def compareProfiles(p1, p2):
```
Comment (ericta-portfolio) [inline backend/compareProfiles.py:1]: wow what a great algorithm

**Review by ericta-portfolio — COMMENTED** [review]
looks good

**Pull request merged**